### ISSUE TITLE
User can't logout / gets automatically logged in.

### ISSUE THREAD

**Issue by mattarderne** [metabase/metabase#10039]
### Bugs

- Your browser and the version: Firefox 66.0.5 (64-bit)
- Your operating system: Mojave
- Your databases: Snowflake
- Metabase version: v0.32.8
- Metabase hosting environment: Docker
- Metabase internal database: Postgres

1. Login to metabase on Firefox 
2. Logout

Expected outcome:
User stays logged out

Actual outcome:
User is logged back in.

Similar experience on Safari, where user is not logged back in unless password is remembered.

Nginx `default.conf` file
```
server {
    listen 80;
    server_name REDACTED;
    return 301 https://REDACTED$request_uri;
}

server {
    listen 443 ssl;
    listen [::]:443 ssl;
    server_name REDACTED;
    charset utf-8;

    ssl on;
    ssl_certificate /etc/REDACTED;
    ssl_certificate_key /etc/REDACTED;

    location / {
        proxy_pass_header Server;
        proxy_set_header Host $http_host;
        proxy_redirect off;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Scheme $scheme;

        proxy_pass http://REDACTED:3000;
    }
}
```
`docker-compose.yml`
```
version: "3.5"

services:
  metabase:
    image: metabase/metabase
    ports:
      - "3000:3000"
    environment:
      - MB_DB_TYPE=postgres
      - MB_DB_DBNAME=metabase
      - MB_DB_PORT=5432
      - MB_DB_USER=${RDS_CLIENT_ID}
      - MB_DB_PASS=${RDS_CLIENT_PWD}
      - MB_DB_HOST=REDACTED  nginx:
    restart: always
    build: nginx
    ports:
      - "80:80"
      - "443:443"
    links:
      - metabase
```

`Dockerfile`
```
FROM tutum/nginx
RUN rm /etc/nginx/sites-enabled/default
ADD default.conf /etc/nginx/sites-enabled/default
WORKDIR /etc/nginx/certs
COPY ./certs /etc/nginx/certs/
```

**Comment by flamber** [comment]
Nice catch - confirmed on JAR without proxy. Not sure how I missed this, but guess I never logout.
But it seems to only be an issue on Firefox.

**Comment by tlrobinson** [comment]
@mattarderne are you using email/password authentication or Google Auth?

**Comment by flamber** [comment]
@tlrobinson I can reproduce this just with a clean local 0.32.8 JAR on H2 with email/password.

**Comment by mattarderne** [comment]
@tlrobinson email/pwd 👍

**Comment by rylonian** [comment]
I wanted to add that this also happens on Internet Explorer.  (I know it is a horrible browser and I try to encourage everyone not to use it, but I just wanted to pass along information.)

**Comment by budthapa** [comment]
Same with Firefox: 67.0 (64-bit), Metabase: v0.32.7, MySQL: 5.7.56 on Ubuntu 18.04.01

**Comment by vietvudanh** [comment]
Can confirm the same problem. 
```
Firefox: 67.0.3 (64-bit)
Metabase: v0.32.8
MacOS 10.14.5
```

**Comment by flamber** [comment]
@tlrobinson 
I've found the reason, but not the cause. It never sends the request `DELETE /api/session` on logout.
Not why it's failing to send that specific request. Seems like it's just disappeared at some point in function `_makeRequest` of `/frontend/src/metabase/lib/api.js`
Think commit caused it: https://github.com/metabase/metabase/commit/060233e36d51861099d85ebae4a75ba9ecd9786e

**Comment by senoadiw** [comment]
Confirming this bug is still present in Metabase version v0.32.9 when accessed with the latest Firefox 67.04 on Win 7 64-bit.

Sign out works fine on Opera 62.0.3331.43.

**Comment by qtamaki** [comment]
Can confirm the same problem.

Firefox: 67.0.4 (64-bit)
Metabase: v0.32.9
Win10 64bit

**Comment by salsakran** [comment]
Hello all, this is fixed in 0.32.10 which just went out.
